src/daemon/utils_cache.c

```diff
@@ -670,7 +670,8 @@ int uc_get_names(char ***ret_names, cdtime_t **ret_times, size_t *ret_number) {
   return 0;
 } /* int uc_get_names */
 
-int uc_get_state(const data_set_t *ds, const value_list_t *vl) {
```

Comment:
We could just remove the `ds` argument. There are only three call sites.

Reply:
done